dhis-2/dhis-web/dhis-web-api-test/src/test/java/org/hisp/dhis/webapi/controller/FileResourceControllerTest.java

```diff
@@ -44,7 +44,8 @@ public void testSaveOrgUnitImage()
             "<<png data>>".getBytes() );
 
         HttpResponse response = POST_MULTIPART( "/fileResources?domain=ORG_UNIT", image );
-        JsonObject savedObject = response.content( HttpStatus.OK ).getObject( "response" ).getObject( "fileResource" );
+        JsonObject savedObject = response.content( HttpStatus.ACCEPTED ).getObject( "response" )
```

Comment:
This needed changing from expecting `OK` to `ACCEPTED` because the response within the controller was inconsistent before. While the `WebMessage` was constructed with `accepted` the response code would be `OK` from the annotation used. This now is consistent again and `ACCEPTED` seemed both the original intent and the more fitting response.